testing/scripts/deploy_staging.sh

```
@@ -0,0 +1,278 @@
+trap cleanup_tmpfiles EXIT
+
+log "Deploying $DRIVER config template..."
+TMPCONF=$(mktemp)
+cp "$TEMPLATE" "$TMPCONF"
+php -r '
+    $f = $argv[1];
+    $c = file_get_contents($f);
+    $c = str_replace("__BOOTSTRAP_ADMIN_USER__", $argv[2], $c);
+    $c = str_replace("__BOOTSTRAP_ADMIN_PASS__", $argv[3], $c);
+    file_put_contents($f, $c);
+' "$TMPCONF" "$BOOT_USER" "$BOOT_PASS"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)